README.rst

```diff
@@ -234,6 +234,93 @@ Below is an example usage of ``nise`` for GCP data::
 
 Generated reports will be generated in daily .csv files with the file format <Report-Prefix>-<Year>-<Month>-<Day>.csv.
 
+======
+nise
+======
+you can install nise with pip::
+    
+    pip install koku-nise
+
+nise is a fake data generator. It allows you to control what is generated.
+An example of what can be generated is found in the example_ocp_static_data.yml file::
+    generators:
+        -OCPGenerator:  # this name is found in the init file of the generator currently
+        start_date: last_month
+        nodes:
+            - node:
+            - node_name: alpha
+            - cpu_cores: 10
+            - memory_gig: 16
+            - resource_id: 55555555 # used to match to AWS resources
+            - cpu_usage:
+                full_period: 1
+            - mem_usage_gig: 
+                full_period: 2
+        
+
+Note: tags in this file are akin to user defined tags in AWS
+Note: If you don't set cost and rate, the generated cost and rate would be random each day.
+
+There example yamls are in the nise repo. 
+
+=============
+List of flags
+=============
+- :code:`--start-date [DATE]` generate data starting on this date, until the end date. If no end date is specified, the current date is the ending date.
+- :code:`--ocp` use this flag if you are creating fake ocp data
+- :code:`--ocp-cluster-id [STRING]` corresponds to the provider resource name. Any string can be used.
+- :code:`--static-report-file [STRING]` the yaml file that will be used to configure nise, for example: :code:`example_ocp_static_data.yml`
+- :code:`--insights-upload [STRING]` the destination directory that nise will store the generated data. For example: :code:`testing/pvc_dir/insights_local`
+
+If you are generating Azure or AWS data, the :code:`--insights-upload` should be set to:
+    - For aws: koku/testing/local_providers/aws_local
+    - For Azure: koku/testing/local_providers/azure_local
+
+=================
+Nise walkthrough
+=================
+This is an example of using nise to generate static report. 
+Run one of the following examples:
+So this creates AWS data from this starting date:
+nise --start-date 2018-06-03
+
+this command creates:
+nise --start-date 2018-06-20 --aws --aws-s3-bucket-name testbucket --aws-s3-report-name cpu_cores
+
+how to create OCP data, and check that it's updated for specific provider:
+nise --start-date 2018-06
+The upload path matches where the data is 
```

Comment:
I think we may want to mention that this path is under koku. In fact we may even want to rename this section to using nise with koku or something like that.